Lib/test/test_importlib/source/test_file_loader.py

```diff
@@ -267,7 +267,7 @@ def test_checked_hash_based_pyc(self):
             )
 
     @util.writes_bytecode_files
-    def test_overriden_checked_hash_based_pyc(self):
+    def test_overridden_checked_hash_based_pyc(self):
```

Comment:
I decided to change function name for the sake of correct spelling. It shouldn't affect any other code and looks like tests aren't failing.

Reply:
I think this is okay since it won't affect anything else, and test can run as it is.